.github/dependabot.yml

```diff
@@ -3,8 +3,8 @@ updates:
 - package-ecosystem: npm
   directory: "/"
   schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+    interval: monthly
+  open-pull-requests-limit: 20
```

Comment:
If it's only happening monthly, is it better to remove the limit altogether?

Reply:
@chadoh quote from docs:
`open-pull-requests-limit`
`By default, Dependabot opens a maximum of five pull requests for version updates. Once there are five open pull requests, new requests are blocked until you merge or close some of the open requests, after which new pull requests can be opened on subsequent updates. Use open-pull-requests-limit to change this limit. This also provides a simple way to temporarily disable version updates for a package manager.`
`This option has no impact on security updates, which have a separate, internal limit of ten open pull requests.`

I think that it should remain, we can have more than 5 updates in 1 month.

Reply:
@chadoh this approach is not documented, but I hope that it will work.
Now we have 2 jobs, 1 will check for updates monthly and another one will do that daily, but it will ignore all dependencies except our won libraries.